pkg/manifests/manifests.go

```diff
@@ -3258,16 +3257,15 @@ func NewSecurityContextConstraints(manifest io.Reader) (*securityv1.SecurityCont
 // It adds "monitoring.openshift.io/name" and "monitoring.openshift.io/hash" labels.
 // Any other labels from the given configmap are discarded.
 //
-// It returns nil, if the given configmap does not contain the "ca-bundle.crt" the data key
+// It returns an error if the given configmap does not contain the "ca-bundle.crt" data key
 // or data is empty string.
-func (f *Factory) HashTrustedCA(caBundleCM *v1.ConfigMap, prefix string) *v1.ConfigMap {
-	caBundle, ok := caBundleCM.Data["ca-bundle.crt"]
-	if !ok || caBundle == "" {
-		// We return here instead of erroring out as we need
```

Comment:
hmm it seems we wrote this at some point, no clue who it was, will this not the be the case anymore?

Reply:
With this change, the operator calls `*caBundleSyncer.syncTrustedCABundle()` which ensures that the resulting config map is populated with the expected keys.